src/components/logo/logo.tsx

```diff
@@ -0,0 +1,15 @@
+// import './logo.css';
+import logoIcon from '../../assets/images/logo.svg';
+
+function logo() {
```

Comment:
use arrow functions

Reply:
name this file as `index.tsx` since this is the entry point for this component.

Reply:
done